src/wowman/curseforge_api.clj

```diff
@@ -41,25 +42,72 @@
         stable-releases (filterv #(= (:releaseType %) stable) latest-files)
 
         ;; no alternative versions, for now
-        stable-releases (remove :exposeAsAlternative stable-releases)
+        stable-releases (remove :exposeAsAlternative stable-releases)]
 
-        ;; I don't know if it's possible, but a group may still have more than one result
-        ;; results are ordered and group-by preserves ordering, so take the first
-        ]
+    ;; I don't know if it's possible, but a group may still have more than one result
+    ;; results are ordered and group-by preserves ordering, so take the first
     (group-by :gameVersionFlavor stable-releases)))
 
+(defn latest-versions-by-gameVersion
+  "given a curseforge-api result, returns a map of release data
+  uses :gameVersion which, if present, indicates the game tracks a release supports
+  prefer this over `latest-versions-by-game-track`"
+  [api-result]
+
+  ;; issue#63: curseforge actually allow a release to be on both retail and classic game tracks
+  ;; the single value under "gameVersionFlavor" is inaccurate and misleading then and we can't trust it
+  ;; instead, we must look at the list "gameVersion" and convert the numbers we find there into game tracks
+  ;; 8.2.0 and 8.2.5 => 'wow_retail'
+  ;; 1.13.2 => 'wow_classic'
+
+  ;; sort latest-files
+  ;; filter stable releases only
+  ;; remove alternate releases
+  ;; create index of latest-files to 
+
+  (let [latest-files (:latestFiles api-result)
+
+        ;; results appear sorted, but lets be sure because we'll be taking the most recent
+        desc (comp - compare) ;; most to least recent (desc)
+        latest-files (sort-by :fileDate desc latest-files)
+
+        ;; stable releases only, for now
+        stable 1 ;; 2 is beta, 3 is alpha
+        stable-releases (filterv #(= (:releaseType %) stable) latest-files)
+
+        ;; no alternative versions, for now
```

Comment:
"alternative" is preferred here: https://grammarist.com/usage/alternate-alternative/

ensure it's applied consistently in codebase